File: C#/TripTalk/TripTalk/Clases/Usuario.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TripTalk.Clases
{
    public class Usuario
    {
        int idUsuario;
        String nombre;
        String apellido;
        String password;
        String correo;
        String rol;
        bool bloqueado;
        public Usuario(){

        }

        public Usuario(int idUsuario, string nombre,string apellido, string correo, string password)
        {
            this.idUsuario = idUsuario;
            this.nombre = nombre;
            this.apellido = apellido;
            this.password = password;
            this.correo = correo;
            rol = "npc";
            bloqueado= false;
        }
        [JsonProperty("idUsuario")]
        public int IdUsuario { get => idUsuario; set => idUsuario = value; }
        [JsonProperty("nombre")]
        public string Nombre { get => nombre; set => nombre = value; }
        [JsonProperty("apellido")]
        public string Apellido { get => apellido; set => apellido = value; }
        [JsonProperty("password")]
        public string Password { get => password; set => password = value; }
        [JsonProperty("correo")]
        public string Correo { get => correo; set => correo = value; }
        [JsonProperty("rol")]
        public string Rol { get => rol; set => rol = value; }

        [JsonProperty("bloqueado")]
        public bool Bloqueado
        {
            get => bloqueado;
            set => bloqueado = value;
        }

        public override string ToString()
        {
            return "Usuario{" + "id_Usuario=" + idUsuario + ", nombre=" + nombre + ", password=" + password + ", correo=" + correo + ", rol=" + rol + '}';
        }
    }
}

File: Java/TripTalk/src/main/java/com/mycompany/triptalk/clases/Usuario.java

package com.mycompany.triptalk.clases;

public class Usuario {
    private int idUsuario;
    private String nombre;
    private String apellido;
    private String password;
    private String correo;
    private String rol;

    public Usuario() {
    }

    public Usuario(int idUsuario, String nombre, String apellido, String password, String correo, String rol) {
        this.idUsuario = idUsuario;
        this.nombre = nombre;
        this.apellido = apellido;
        this.password = password;
        this.correo = correo;
        this.rol = rol;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getApellido() {
        return apellido;
    }

    public void setApellido(String apellido) {
        this.apellido = apellido;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    public String getCorreo() {
        return correo;
    }

    public void setCorreo(String correo) {
        this.correo = correo;
    }

    public String getRol() {
        return rol;
    }

    public void setRol(String rol) {
        this.rol = rol;
    }

    public int getIdUsuario() {
        return idUsuario;
    }

    public void setIdUsuario(int idUsuario) {
        this.idUsuario = idUsuario;
    }
    
    
    
}
